bgcval2/download_from_mass.py

```diff
@@ -602,43 +622,24 @@ def deleteBadLinksAndZeroSize(outputFold, jobID):
 
     #####
     # All yearly files
-    if keys == []:
-        downloadMass(jobID)
 
-    elif 'noMoo' in keys:
-        downloadMass(jobID, doMoo=False)
+    if keys == []:
+        download_from_mass(jobID, doMoo=True)
+        return
```

Comment:
think there's no need for an empty return here

Reply:
The return skips the rest of the keys.